Worker node req Essential vs Enterpr (#4521)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/prerequisites/index.md

## Worker nodes

> For Enterprise, the recommendation is at least four worker nodes. The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.  (See below for minimal suggested requirements)

Reviewer: Even for essential you need four worker nodes...

```suggestion
For DKP Enterprise or Essential, the recommendation is at least four worker nodes. The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.  (See below for minimal suggested requirements)
```

> For Enterprise, the recommendation is at least four worker nodes. Essential, however, only requires one worker node.  The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.  (See below for minimal suggested requirements)

Reviewer: ```suggestion
The minimum number of worker nodes for an Essential or Enterprise cluster requires at least four worker nodes.  The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.  (See below for minimal suggested requirements)
```
